apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsTextControl.tsx

```diff
@@ -50,7 +50,9 @@ export function JsonFormsTextControl({
   return (
     <Box mt="1.25rem" _first={{ mt: 0 }}>
       <FormControl isRequired={required} isInvalid={!!errors}>
-        <FormLabel description={description}>{label}</FormLabel>
+        <FormLabel description={description} margin="0rem 0rem 0.5rem 0rem">
```

Comment:
super ultra nit: we can use the shorthand `mb` (margin bottom). similar conventinos exist for `top/bototm/left/right` and `mx/my` for vertical.

see [here](https://v2.chakra-ui.com/docs/styled-system/style-props#margin-and-padding)

Reply:
@seaerchin in this case im overwritting the margins from FormLabel from OGP design system. 

So will `mt="0rem" mr="0rem" mb="0.5rem" ml="0rem"` be better?

Reply:
wait my bad, i thought the default margins were 0! if you're overwriting all 4, please just stick with this and ignore me

Reply:
default is mr16 mb16

but i thought to just put a 0 for everything instead of mr0 mb8 

otherwise someone reading it might be left thinking whats mt and ml when debugging in the future